classes/notification/managerDelegate/EditorialReportNotificationManager.inc.php

```diff
@@ -47,9 +47,12 @@ public function __construct(int $notificationType) {
 	public function initialize(Context $context, DateTimeInterface $dateStart, DateTimeInterface $dateEnd) : void
 	{
 		$this->_context = $context;
+		$locale = $this->_context->getPrimaryLocale();
 		$dateStart = $dateStart;
 		$dateEnd = $dateEnd;
 
+		\AppLocale::requireComponents([LOCALE_COMPONENT_PKP_MANAGER, LOCALE_COMPONENT_APP_MANAGER, LOCALE_COMPONENT_PKP_USER], $locale);
+
```

Comment:
Ah, the `requireComponents` has a "Backwards compatibility" message about this syntax using an array of components. Is there an intention to remove it?
I've used it because I think it looks clearer and safer (`$locale == null` would fail).

Reply:
No, I've got no intention of removing that syntax (and if I remember right, I just wanted to avoid typing `array(`...`)` a thousand times -- this predates the `[]` syntax in PHP). You're OK to use this.